pkg/cmd/server/rhoai-normalizer/controller.go

```diff
@@ -361,24 +361,22 @@ func (r *RHOAINormalizerReconcile) processKFMR(ctx context.Context, name types.N
 					if se.Name != nil && *se.Name == is.Namespace {
 						// FOUND the match !!
 						// reminder based on explanations about model artifact actually being the "root" of their model, and what has been observed in testing,
-						// the ID for the registered model and model artifact appear to match
```

Comment:
yeah this proved to be an oversimplification / wrong once tested with different models